client/src/state/campaign/CampaignReducer.js

```diff
@@ -1,12 +1,16 @@
+/* global window */
 import deepFreeze from 'deep-freeze-strict';
 import ACTION_TYPES from './CampaignActionTypes';
 
+// retrieve showMessage state from user's localStorage
+const showMessage = window.localStorage.getItem('campaign.showMessage');
 const initialState = deepFreeze({
   campaignId: null,
   changeSetItemId: null,
   isPublishing: false,
   view: null,
   newItem: null,
+  showMessage: showMessage === null,
```

Comment:
We should only show message if showMessage == null && there are no campaigns; I have campaigns loaded, and my initial load displayed the help message when I didn't expect it.

Reply:
To elaborate:
The trouble with adding the logic to check for campaigns - the `showMessage` state becomes ternary and ambiguous until the list of Campaign records has been received, meaning a few extra layers for something that is stored in `localStorage` after the first time it's closed anyway.

P.S. Great that we agree that it's not that important.